### PR TITLE
Add basic version of new map

### PR DESCRIPTION
**Pull request by toberge**
- [x] Terrain with mountains and a valley for the arena
- [x] Basic textures
- [x] Basic terrain shader
- [x] Whiteboxed buildings
- [x] Multiple spawnpoints, shuffled

## Progress images:

image

image

### CODE REVIEW

File: Assets/Scripts/Gamestate/MatchController.cs [hunk 1]
```diff
@@ -148,7 +148,7 @@ public IEnumerator WaitAndStartNextRound()
         yield return new WaitForSeconds(roundEndDelay);
         // This needs to be called after inputs are set at start the first time this is needed.
         PlayerInputManagerController.Singleton.ChangeInputMaps("FPS");
-        SceneManager.LoadScene("DemoArena");
+        SceneManager.LoadScene("Cratertown");
```
Comment (Fueredoriku) [inline Assets/Scripts/Gamestate/MatchController.cs:151]: Maybe we should rename this to PascalCase?

File: Assets/Scripts/UI/StatBar.cs [hunk 2]
```diff
@@ -66,6 +66,14 @@ void Awake()
 
     private void UpdateMeter()
     {
+        // Avoid one peculiar bug where the RectTransform is null
+        // and the MissingReferenceException spam prevents the next scene from loading
+        if (baseRect == null)
```
Comment (Fueredoriku) [inline Assets/Scripts/UI/StatBar.cs:71]: Seems very weird, is there something underlying here or is this just Unity UI jank (tm)?

**Review by Fueredoriku — COMMENTED** [review]
Very cool! 😎 A nice shuffle algorithm implementation too!

I only have a couple of nitpicks in the review-comments.

Also, I think we should implement a barrier to prevent this, although that can be handled as it's own PR to resolve #92 :
image

File: Assets/Scripts/Gamestate/MatchController.cs (hunk 1 above)
Comment (toberge) [reply Assets/Scripts/Gamestate/MatchController.cs:151]: Yeah I thought of it but forgor to do it :skull:

File: Assets/Scripts/UI/StatBar.cs (hunk 2 above)
Comment (toberge) [reply Assets/Scripts/UI/StatBar.cs:71]: I suspect there's an underlying issue, though this is outside the scope of this PR so I just slammed inn a temporary fix :upside_down_face:

Comment (Fueredoriku) [reply Assets/Scripts/UI/StatBar.cs:71]: This isssue is tied to #70 , so we'll keep toss this on there.

**Review by Fueredoriku — APPROVED** [review]
Read code and tested, works on my machine 🎉

**Pull request merged**